homepage/models.py

```diff
@@ -122,6 +134,17 @@ class FollowedUserCourses(models.Model):
     def __str__(self):
         return f'user = {self.user}, course = {self.course}'
 
+    @staticmethod
+    def get_courses_followed_by_appUser(appUser):
+        pairs = FollowedUserCourses.objects.filter(user=appUser)
+        followed_appUser_courses = list()
+
+        # get only 'course' elements from user_course_pair elements
+        for user_course_pair in pairs:
+            followed_appUser_courses.append(user_course_pair.course)
+
+        return followed_appUser_courses

```

Comment:
in python you have the List Comprehension that doing this task in one line like:
```suggestion
        # get only 'course' elements from user_course_pair elements
        return [ user_course_pair.course for user_course_pair in pairs ]

```

Reply:
changed